### PR TITLE
Feature/napsspo 514

### PR DESCRIPTION
**Pull request by kharyam**
Tekton ClusterTask for buildah

### CODE REVIEW

File: tekton/tasks/tssc-create-container-image.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      PATH=/opt/rh/rh-python36/root/usr/bin:${PATH}
+
+      # Persist the built image in the pvc for subsequent steps (e.g., pushes)
+      mkdir $(workspaces.tssc-venv.path)/containers || true
+      rm -fr /var/lib/containers
+      ln -s $(workspaces.tssc-venv.path)/containers /var/lib/containers
+
+      source $(workspaces.tssc-venv.path)/tssc/bin/activate
+      python -m tssc -c cicd/tssc-config.yml -s create-container-image
+      echo "DEBUG - print results"
```
Comment (kfrankli) [inline tekton/tasks/tssc-create-container-image.yml:34]: Should this be here?

Comment (kharyam) [reply tekton/tasks/tssc-create-container-image.yml:34]: Yep, this is consistent with the other clustertasks. Is there a specific line that's questionable?

Comment (itewk) [reply tekton/tasks/tssc-create-container-image.yml:34]: the echo DEBUG lines can be taken out now that the python library auto outputs the results.

Comment (kharyam) [reply tekton/tasks/tssc-create-container-image.yml]: Updated

File: tekton/java-backend/pipeline.yml [hunk 2]
```diff
@@ -7,39 +7,38 @@ spec:
...
+      - name: tssc-venv
+        workspace: local-tssc-venv
+    - name: generate-metadata
+      resources:
       taskRef:
         kind: ClusterTask
         name: no-op
     - name: tag-source-code-with-metadata
       resources:
-        inputs:
```
Comment (kfrankli) [inline tekton/java-backend/pipeline.yml:20]: So what this only needed for setup?

**Pull request merged**